First spec of the proto file

felixmercermoss opened the pull request.

Bettiher commented on rubus.proto:
```diff
@@ -0,0 +1,82 @@
+
+message RecommendedItems {
+
+    repeated string uri = 1;
```
We're not supposed to give back some "version"? On how Rubus made the recommendations?

tatiana replied:
Well spotted! We completely forgot about it - adding now

Bettiher commented on rubus.proto:
```diff
@@ -0,0 +1,82 @@
+
+// REQUESTS
+
+message ListRecommendationsRequest {
```
Is Bramble supposed to pass the characteristic(s) the User Profile is gonna be built on as well? Or is it defined only in Rubus (i.e. master brand, or genre level 2 etc.)? Open question here

tatiana replied:
That's a great point, @Bettiher, since Snowdrop will be enriching each item anyway, it seems we only need to give to Bramble PID and item provenance for now - what do you think?
If this changes in future, we can add more info when it is needed.

Bettiher replied:
That wasn't my question. Sorry I wasn't clear. 
I am talking about the property we're gonna use to do the aggregation (i.e. the user profiles), that Rubus is gonna pass on to Amanita to get the RECs

felixmercermoss replied:
Sure we could make so that we specify the list of properties that we use to generate the user profiles but it's not clear at the moment where Bramble would get that information (probably the Bramble config file). As that hasn't been implemented yet, I suggest we keep it simple by keeping the logic in Rubus for now. Unless you have any strong objections?

Bettiher replied:
I am fine with that.

tatiana commented on rubus.proto:
```diff
@@ -0,0 +1,82 @@
+enum MediaTypeParam {
+    VIDEO = 0;
+    AUDIO = 1;
+    TEXT = 3;
```
Thanks, @maria-panteli! Excellent feedback. We had opied this mistake from Bramble 🙈 

We're fixing in both places.

felixmercermoss commented:
DATALAB-266: First changes to implement gRPC. 
- Includes reviewed spec for proto file
- Updated readme
- Removed http legacy
- Updated requirements
- Updated Dockerfile
- Updated Tests (none implemented)
